hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -0,0 +1,228 @@
+        accept_data = super(MayaPublishGeometryPlugin, self).accept(task_settings, item)
+        if not accept_data.get("accepted"):
+            return accept_data
+
+        # check that the AbcExport command is available!
+        if not mel.eval("exists \"AbcExport\""):
+            self.logger.debug(
+                "Item not accepted because alembic export command 'AbcExport' "
+                "is not available. Perhaps the plugin is not enabled?"
+            )
+            accept_data["accepted"] = False
+
+        # TODO: if already exported geo exists, don't re-export
+
+        # return the accepted info
+        return accept_data
+
+
+    def validate(self, task_settings, item):
+        """
+        Validates the given item to check that it is ok to publish. Returns a
+        boolean to indicate validity.
+
+        :param task_settings: Dictionary of Settings. The keys are strings, matching
+            the keys returned in the settings property. The values are `Setting`
+            instances.
+        :param item: Item to process
+        :returns: True if item is valid, False otherwise.
+        """
+        # check that there is still geometry in the scene:
+        if not cmds.ls(geometry=True, noIntermediate=True):
+            error_msg = (
+                "Validation failed because there is no geometry in the scene "
+                "to be exported. You can uncheck this plugin or create "
+                "geometry to export to avoid this error."
+            )
+            self.logger.error(error_msg)
+            raise TankError(error_msg)
+
+        return super(MayaPublishGeometryPlugin, self).validate(task_settings, item)
+
+
+    def publish_files(self, task_settings, item, publish_path):
+        """
+        Overrides the inherited method for copying the work file to the publish location
+        to instead export out the scene geometry to the publish_path location.
+
+        :param task_settings: Dictionary of Settings. The keys are strings, matching
+            the keys returned in the settings property. The values are `Setting`
+            instances.
+        :param item: Item to process
+        :param publish_path: The output path to publish files to
+        """
+        publisher = self.parent
+
+        # set the alembic args that make the most sense when working with Mari.
+        # These flags will ensure the export of an Alembic file that contains
+        # all visible geometry from the current scene together with UV's and
+        # face sets for use in Mari.
+        alembic_args = [
+            # only renderable objects (visible and not templated)
+            "-renderableOnly",
+            # write shading group set assignments (Maya 2015+)
+            "-writeFaceSets",
+            # write uv's (only the current uv set gets written)
+            "-uvWrite",
+            # apply euler filter to avoid gimbal lock issues
+            "-eulerFilter"
+        ]
```

Comment:
We should keep these consistent with `export_geo` or maybe have similar settings provided in the plugin?

Reply:
Yes, exposing the same as task settings now, and on UI.